src/pages/LogInRegister.tsx

```diff
@@ -0,0 +1,28 @@
+import '../assets/css/LogInRegister.css';
+
+//Components imports
```

Comment:
No need for you to add these type of comments, 100% of imports go at the beginning of the file, so no need to add a comment 😉

Reply:
I see, thank you so much for your feedback! I uploaded the different images into the different PR. Now I will fix every point I replied to you in this PR. Which is for you the best way to continue, to keep working on the same branch to keep it all in the same PR or to create another one and have different PR?

Thank you so much, as I am not 100% clear which is the best option I'll take a look to the other comments before to continue working on this PR

Reply:
Hi @JorgeHBatista!
Sorry for the late reply. I've been on vacation last week. You can do the fixes in this branch, the we merge after approval, and then you can keep the other PRs to merge to main, as this code will be already in main.
Take into account that with the latest changes in main, you'll have some conflicts to fix in your branch